test/lu.jl

```diff
@@ -1,8 +1,9 @@
-using StaticArrays, Compat.Test
+using StaticArrays, Test, LinearAlgebra
 
+@testset "LU decomposition" begin
```

Comment:
This was removed specifically to have more stuff printed, since we were running into travis timeouts. Other changes were made at the time to reduce test time, so maybe this is OK, but I'm just a little worried about travis flakiness.